Document/0x06c-Reverse-Engineering-and-Tampering.md

```diff
@@ -438,23 +438,24 @@ Next, navigate to a new website in Safari. You should see traced function calls
 
 #### Emulation-based Analysis
 
-Emulation is imitation of a certain computer platform or program on another platform or program. The software or hardware performing this imitation is called an _emulator_. Emulator provides a much cheaper alternatives to an actual device, where a user can manipulate it without worrying about damaging the device. As discussed in "[Emulation-based analysis](0x05c-reverse-engineering-and-tampering#emulation-based-analysis "Emulation-based analysis")" section, there are multiple emulators available for Android, but unlike Android, for iOS practically there are practically no viable emulators available.
-
 ##### iOS Simulator
 
-Apple provides a simulator app within Xcode which provides an user interface for iPhone, iPad or Apple Watch. As per Apple documentation, "simulator allows you to rapidly prototype and test builds of your app during the development process." Although the iOS simulator provides a similar look and feel as a real iOS device and also allows to run a debug application, but actually it is not an emulator. Difference between a simulator and an emulator often causes confusion and leads to use of the two terms interchangeably, but in reality they are different, specially for our current use case.
+Apple provides a simulator app within Xcode which provides a _real iOS device looking_ user interface for iPhone, iPad or Apple Watch. It allows you to rapidly prototype and test debug builds of your applications during the development process, but actually **it is not an emulator**.
+
+The difference between a simulator and an emulator often causes confusion and leads to use of the two terms interchangeably, but in reality they are different, specially for the iOS use case. An emulator mimics both the software and hardware environment of a targeted platform. On the other hand, a simulator only mimics the software environment.
```

Comment:
I think I'd actually take from here 445 until 449 and put this explanation in 0x4c as well. It's very nice to have it in one place talking about both platforms and to keep this section short. That would leave us here with only 2 nice paragraphs going straight to the point ;)